### PR TITLE
Upgraded to SDK 38 and fixed typos in LICENSE

### PR DESCRIPTION
**Pull request by saivittalb**
Fixes issue #145 and also corrects the LICENSE file by adding the name of the author, year, and project name.

Though the CI build is failing, the app works perfectly on both iOS and Android. Also, I think the CI script needs to be modified to accommodate this change.

### CODE REVIEW

File: app.example.json [hunk 1]
```diff
@@ -29,7 +34,7 @@
           "config": {
             "organization": "nmf",
             "project": "not-my-fault-earth",
-            "authToken": null
+            "authToken": {}
```
Comment (PierreBresson) [inline app.example.json:37]: leave it null

File: package.json [hunk 2]
```diff
@@ -24,9 +24,9 @@
     ]
   },
   "dependencies": {
-    "@expo/vector-icons": "10.1.0",
-    "@react-native-community/datetimepicker": "2.3.2",
-    "@react-native-community/masked-view": "0.1.6",
+    "@expo/vector-icons": "^10.0.0",
```
Comment (PierreBresson) [inline package.json:27]: Please do not put downgrade any package.

File: package.json [hunk 3]
```diff
@@ -37,12 +37,12 @@
     "@types/react-redux": "7.1.7",
     "@types/webpack-env": "1.15.0",
     "carbon-footprint": "1.3.4",
-    "expo": "37.0.10",
-    "expo-asset": "8.1.5",
-    "expo-constants": "9.0.0",
-    "expo-font": "8.1.1",
-    "expo-localization": "8.1.0",
-    "expo-web-browser": "8.2.1",
+    "expo": "^38.0.0",
```
Comment (PierreBresson) [inline package.json:40]: Please do not put any ^ so we all have the same versions

File: package.json [hunk 4]
```diff
@@ -37,12 +37,12 @@
...
+    "expo-asset": "~8.1.7",
+    "expo-constants": "~9.1.1",
+    "expo-font": "~8.2.1",
```
Comment (PierreBresson) [inline package.json:43]: No ~ as well

File: package.json (hunk 2 above)
Comment (saivittalb) [reply package.json]: I've restored the original version of this dependency. But I've downgraded one other dependency due to the reason stated in the comment below

**Comment by saivittalb** [comment]
> @saivittalb thanks for having a look!
> CI is working properly and does not need to be modified since we are just updating expo. I cannot run the app on my computer for the moment.

Hey, so I've made changes according to the guidelines requested. I had to degrade ```sentry-expo``` dependency due to an error after making the required changes. The issue I faced with ```sentry-expo``` is referenced [here](https://github.com/expo/sentry-expo/issues/120).

The CI script is fine. I didn't showcase what the issue with it was exactly. I'm attaching a screenshot below which helps my case. So, when I committed using git, this error popped up, and the same error causes the CI build to fail. Since I didn't play with the code and only updated the Expo SDK, I think something has to be done.

Error log

###### Note

This is a pre-commit husky error popping up when ```git commit -m <"Message">``` is run on my Mac and the same thing can be found in CI logs as well without husky pre-commit.

Apart from this, the app is working perfectly. I'm attaching a screenshot captured when I ran the app after making the changes.

Simulator Screen Shot - iPhone SE (2nd generation) - 2020-07-12 at 03 37 56

**Comment by saivittalb** [comment]
Hey, I think I need a little help in fixing that script. Meanwhile, I'll try myself.

**Comment by PierreBresson** [comment]
> Hey, I think I need a little help in fixing that script. Meanwhile, I'll try it myself.

@saivittalb  No problem, I'm gonna have a look.

**Comment by PierreBresson** [comment]
@saivittalb the following code should fix the error :
```
const getCreationDate: (Emission) => string = prop("creationDate");

const getLatestEmission = (emissions: Array<Emission>) =>
  isNilOrEmpty(emissions)
    ? null
    : reduce(maxBy(getCreationDate), emissions[0], emissions);
```

**Comment by saivittalb** [comment]
@PierreBresson Thanks for the code. I was able to fix that error. After that, I had to update test suites and add ```expo-splash-screen``` to run all the test suites successfully.

I'm attaching the screenshots of the app after a successful run.

Simulator Screen Shot - iPhone SE (2nd generation) - 2020-07-25 at 17 41 56
Simulator Screen Shot - iPhone SE (2nd generation) - 2020-07-25 at 18 13 47

**Comment by PierreBresson** [comment]
Interesting fact : for me, the app was blocked on the splash screen because of expo. I was getting `None of these files exist:   * debugger- warn Error: not opened` when trying to debug. I fixit it by running locally `npm i -g expo-cli` and then `expo upgrade`. I think I would have get the same error if I had deleted the expo app on the ios simulator.

**Comment by saivittalb** [comment]
> Interesting fact : for me, the app was blocked on the splash screen because of expo. I was getting `None of these files exist: * debugger- warn Error: not opened` when trying to debug. I fixit it by running locally `npm i -g expo-cli` and then `expo upgrade`. I think I would have get the same error if I had deleted the expo app on the ios simulator.

Yeah. The latest Expo CLI version was released just 11 days ago and it's currently at `3.22.3`. Even I faced some issues while running the app. I was on `3.22.1`, and upgrading did the trick instantly. 🎉

**Pull request merged**